svea_vision/launch/zed_main.launch

```diff
@@ -17,9 +17,10 @@
     <arg name="depth_image"     value="/zed/zed_node/depth/depth_registered"/>
     <arg name="image"           value="/zed/zed_node/rgb/image_rect_color"/>
 
+    <arg name="map_img_name" default="$(find svea_core)/maps/sml.yaml"/>
+    <node pkg="map_server" name="map_server" type="map_server" args="$(arg map_img_name)"/>
```

Comment:
* Rename `map_img_name` to `map_path`
* Add conditional for `map_server`: `<node if="$(eval map_path)" pkg="map_server" .../>`

You might have to do `$(eval bool(map_path))`, `$(eval map_path != '')` or similar